### PR TITLE
[NEW] Keyboard shortcuts

### LINKED ISSUE #812: [NEW] Support keyboard shortcuts
# Description

We should have some keyboard shortcuts in the app, this is super used by iPad users with external keyboards.

Some ideas:
- When looking into the subscriptions list, `⌘ + F` should focus on the search;
- When looking into the chat list, `⌘ + I` should open channel information;

### PR DESCRIPTION
**Pull request by cardoso**
@RocketChat/ios 

Shortcuts so far:

Closes #812

### CODE REVIEW

File: Rocket.Chat/Controllers/Base/Main/MainSplitViewControllerShortcuts.swift [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+    }
+
+    @objc func shortcutScrollMessages(_ command: UIKeyCommand) {
+        guard
+            !isPresenting,
+            let input = command.input,
+            let offset = chatViewController?.collectionView?.contentOffset,
+            let maxHeight = chatViewController?.collectionView?.contentSize.height,
+            let collectionView = chatViewController?.collectionView
+            else {
```
Comment (filipealva) [inline Rocket.Chat/Controllers/Base/Main/MainSplitViewControllerShortcuts.swift]: little indentation issue on the `else` statement

Comment (cardoso) [reply Rocket.Chat/Controllers/Base/Main/MainSplitViewControllerShortcuts.swift]: Thanks!

**Comment by cardoso** [comment]
Done @rafaelks :+1:

File: Rocket.Chat/Controllers/Base/Main/MainSplitViewControllerShortcuts.swift [hunk 2]
```diff
@@ -127,14 +127,26 @@ extension MainSplitViewController {
...
     @objc func shortcutRoomActions(_ command: UIKeyCommand) {
+        guard chatViewController?.subscription != nil else {
+            return
+        }
+
         chatViewController?.toggleActions()
     }
 
     @objc func shortcutRoomMessageSearch(_ command: UIKeyCommand) {
+        guard chatViewController?.subscription != nil else {
```
Comment (rafaelks) [inline Rocket.Chat/Controllers/Base/Main/MainSplitViewControllerShortcuts.swift]: Let's use `validated()` here?

File: Rocket.Chat/Controllers/Base/Main/MainSplitViewControllerShortcuts.swift [hunk 3]
```diff
@@ -127,14 +127,26 @@ extension MainSplitViewController {
...
     @objc func shortcutUpload(_ command: UIKeyCommand) {
+        guard chatViewController?.subscription != nil else {
+            return
+        }
+
         chatViewController?.toggleUpload()
     }
 
     @objc func shortcutRoomActions(_ command: UIKeyCommand) {
+        guard chatViewController?.subscription != nil else {
```
Comment (rafaelks) [inline Rocket.Chat/Controllers/Base/Main/MainSplitViewControllerShortcuts.swift]: Let's use `validated()` here?

File: Rocket.Chat/Controllers/Base/Main/MainSplitViewControllerShortcuts.swift [hunk 4]
```diff
@@ -127,14 +127,26 @@ extension MainSplitViewController {
     }
 
     @objc func shortcutUpload(_ command: UIKeyCommand) {
+        guard chatViewController?.subscription != nil else {
```
Comment (rafaelks) [inline Rocket.Chat/Controllers/Base/Main/MainSplitViewControllerShortcuts.swift]: Let's use `validated()` here?

**Comment by rafaelks** [comment]
@cardoso Just few things and the conflicts to be resolved! 👍

**Pull request merged**